exist-versioning-release.md

```diff
@@ -150,48 +231,72 @@ You will require a system with:
+            <activeProfile>existdb-release-signing</activeProfile>
+        </activeProfiles>
     </settings>
+```
+
+1. Merge any outstanding PRs that have been reviewed and accepted for the milestone eXist-5.3.0.
+
+2. Create a new clone of the branch you want to release from into a new folder.
+   In the example below the `develop` branch is cloned into the folder `release`
+
+
+    ```bash
+    git clone \ 
+      -b develop \
+      git@github.com:eXist-db/exist.git \
+      release
+    ```
+    
+3. switch to that directory
+
+    ```bash
+    cd release
     ```
 
-2. You will need your GPG Key and Java KeyStore credentials for signing the release artifacts in the `<activeProfiles`> section on your machine in your local `~/.m2/settings.xml` file, e.g.:
-    ```xml
-    <profiles>
-   
-       <profile>
-           <id>existdb-release-signing</id>
-           <properties>
-               <existdb.release.key>ABC1234</existdb.release.key>
-               <existdb.release.public-keyfile>${user.home}/.gnupg/pubring.gpg</existdb.release.public-keyfile>
-               <existdb.release.private-keyfile>${user.home}/.gnupg/secring.gpg</existdb.release.private-keyfile>
-               <existdb.release.key.passphrase>your-password</existdb.release.key.passphrase>
-   
-               <existdb.release.keystore>${user.home}/your.store</existdb.release.keystore>
-               <existdb.release.keystore.pass>your-keystore-password</existdb.release.keystore.pass>
-               <existdb.release.keystore.key.alias>your-alias</existdb.release.keystore.key.alias>
-               <existdb.release.keystore.key.pass>your-key-password</existdb.release.keystore.key.pass>
-           </properties>
-       </profile>
-   
-    </profiles>
-
-
-    <activeProfiles>
-   
-           <activeProfile>existdb-release-signing</activeProfile>
-   
-    </activeProfiles>
+    You can verify that origin now points to the main repository `git@github.com:eXist-db/exist.git`
+    
+    ```bash
+    git remote -v
     ```
 
-3.  Merge any outstanding PRs that have been reviewed and accepted for the milestone eXist-5.3.0.
 
-4.  Make sure that you have the HEAD of `origin/develop` (or `upstream` if you are on a fork).
 
-5.  Prepare the release, if you wish you can do a dry-run first by specifiying `-DdryRun=true`:
-    ```
-    $ mvn -Ddocker=true -Dmac-signing=true -Dizpack-signing=true -Darguments="-Ddocker=true -Dmac-signing=true -Dizpack-signing=true" release:prepare
+4.  Prepare the release
+    **NOTE:** If this is the first time you prepare a release or you made changes to the process you should do a dry-run first by setting `-DdryRun=true`
+    With `-V` you can also check the maven and JDK version that will be used
+
+    ```bash
+    mvn \
+      -Pdocker \
+      -Pinstaller \
+      -Pconcurrency-stress-tests \
+      -Pmicro-benchmarks \
+      -Darguments="-Ddocker=true -Dmac-signing=true -Dizpack-signing=true -Dmac.codesign.identity=\"your codesigning ID\" \
```

Comment:
The `-P` args should likely also appear in `-Darguments`

Reply:
The codesign identity could be moved into `settings.xml` which would simplify this.

Reply:
As noted down below the duplication is not necessary.
Would you be willing to share the snippet how to set the codes signing identity in settings.xml?